<?xml version="1.0"?>
<launch>
    <arg name="dir" default="0"/>
    <param name="runDirection" value="$(arg dir)"/>
    <node pkg="cra_traffic_sim" type="anl_planner_tracker.py" name="highlevel_tracker" output="screen"/>
</launch>